lib/modules/datasource/go/base.spec.ts

```diff
@@ -345,23 +347,17 @@ describe('modules/datasource/go/base', () => {
         });
       });
 
-      it('handles go-import with azure devops source', async () => {
-        const meta =
-          '<meta name="go-import" content="dev.azure.com/my-organization/my-project/_git/my-repo.git git https://dev.azure.com/my-organization/my-project/_git/my-repo.git" />';
+      it('returns null for invalid azure devops source', async () => {
         httpMock
           .scope('https://dev.azure.com')
-          .get('/my-organization/my-project/_git/my-repo.git?go-get=1')
-          .reply(200, meta);
+          .get('/foo/bar.git?go-get=1')
+          .reply(200);
```

Comment:
Would Azure really return a 200 in this case?

Reply:
That depends on the input value, if `foo` is your organization name, `bar.git` a valid project name and the token can access it, then it will response with a 200. But for anything else it will result in a 401.
When I change the mock reply to 401 the test will fail with:
```
 FAIL  lib/modules/datasource/go/base.spec.ts (219 MB heap size)
  ● modules/datasource/go/base › go-get requests › meta name=go-source › returns null for invalid azure devops source

    HTTPError: Response code 401 (Unauthorized)

      at Request.<anonymous> (node_modules/.pnpm/got@11.8.6/node_modules/got/dist/source/as-promise/index.js:118:42)
```
I usually not coding in typescript and have no expirience with the used testing framework to have any clue how to solve it.